packages/waggle/src/libp2p/connectionsManager.ts

```diff
@@ -201,12 +206,18 @@ export class ConnectionsManager {
 
     libp2p.connectionManager.on('peer:connect', (connection: Connection) => {
       log(`${peerId.toB58String()} connected to ${connection.remotePeer.toB58String()}`)
+      this.connectedPeers.add(connection.remotePeer.toB58String())
+      this.emit('peer:connect', {
```

Comment:
Won't this conflict with the 'peer:connect' event above (the same name)? I'm not sure, just asking.

Reply:
seems to be no conflict